### DISCUSSION TITLE
Screen capture doesn't work properly when the taskbar is placed on the left

### DISCUSSION THREAD

**Discussion by kwencel** [apohl79/audiogridder#646] [Help]
I have the taskbar docked at the left side of the screen instead of the default bottom position.

I see that AudioGridder tries to open every plugin UI at position (0,0), which is the top-left corner of the screen. However, it does not take into consideration that I have a taskbar there. As a result, part of the plugin UI is covered by the taskbar and cannot be controlled remotely.

I think one of the fixes could be:
1) Taking the taskbar location into consideration when placing the plugin UI and setting a screen capture region.
2) Going with the virtual desktop approach, where the taskbar probably does not exist, so it won't be an issue.

I assume the same thing happens when someone positions the taskbar at the top, but I didn't test it.

I am using AudioGridder 1.2 Beta 3 and RDP.

**Comment by apohl79** [comment]
thx for reporting this. created a bug issue.
